noetic-foxy/package.xml

```diff
@@ -84,80 +122,38 @@
   <depend>libqt5-opengl</depend>
   <depend>libqt5-opengl-dev</depend>
   <depend>libqt5-widgets</depend>
-  <depend>libssh2</depend>
-  <depend>libx11-dev</depend>
-  <depend>libxi-dev</depend>
-  <depend>libxmu-dev</depend>
-  <depend>linux-kernel-headers</depend>
   <depend>message_filters</depend>
-  <depend>message_generation</depend>
-  <depend>message_runtime</depend>
   <depend>nav_msgs</depend>
   <depend>nmea_msgs</depend>
-  <depend>nodelet</depend>
-  <depend>opengl</depend>
   <depend>pcl_conversions</depend>
   <depend>pcl_msgs</depend>
   <depend>pcl_ros</depend>
-  <depend>position_controllers</depend>
-  <depend>python-imaging</depend>
   <depend>python3-nose</depend>
-  <depend>python_qt_binding</depend>
   <depend>python3-requests</depend>
-  <depend>python3-rospkg</depend>
   <depend>python3-serial</depend>
   <depend>python3-yaml</depend>
   <depend>qtbase5-dev</depend>
   <depend>robot_state_publisher</depend>
-  <depend>rosbag</depend>
-  <depend>rosbag_storage</depend>
   <depend>rosbridge_server</depend>
-  <depend>rosconsole</depend>
-  <depend>roscpp</depend>
-  <depend>roscpp_serialization</depend>
-  <depend>rosgraph_msgs</depend>
-  <depend>roslaunch</depend>
-  <depend>roslib</depend>
-  <depend>roslint</depend>
-  <depend>rospy</depend>
-  <depend>rostest</depend>
-  <depend>rosunit</depend>
-  <depend>rqt_gui</depend>
-  <depend>rqt_gui_py</depend>
-  <depend>rqt_plot</depend>
-  <depend>rviz</depend>
   <depend>sensor_msgs</depend>
-  <depend>socketcan_bridge</depend>
-  <depend>socketcan_interface</depend>
-  <depend>sound_play</depend>
   <depend>std_msgs</depend>
   <depend>std_srvs</depend>
   <depend>stereo_msgs</depend>
-  <depend>swri_math_util</depend>
-  <depend>swri_roscpp</depend>
-  <depend>swri_serial_util</depend>
-  <depend>swri_string_util</depend>
-  <depend>swri_nodelet</depend>
-  <depend>tablet_socket_msgs</depend>
   <depend>tf2</depend>
   <depend>tf2_geometry_msgs</depend>
   <depend>tf2_msgs</depend>
   <depend>tf2_ros</depend>
-  <depend>tf_conversions</depend>
-  <depend>tf</depend>
   <depend>tinyxml</depend>
-  <depend>topic_tools</depend>
-  <depend>transmission_interface</depend>
   <depend>v4l-utils</depend>
-  <depend>vector_map_msgs</depend>
   <depend>velocity_controllers</depend>
   <depend>velodyne</depend>
-  <depend>velodyne_description</depend>
-  <depend>velodyne_gazebo_plugins</depend>
   <depend>velodyne_pointcloud</depend>
-  <depend>velodyne_pcl</depend>
   <depend>visualization_msgs</depend>
   <depend>xacro</depend>
-  <depend>xmlrpcpp</depend>
   <depend>yaml-cpp</depend>
-</package>
+
+  <export>
+    <build_type condition="$ROS_VERSION == 1">catkin</build_type>
+    <build_type condition="$ROS_VERSION == 2">ament_cmake</build_type>
```

Comment:
Since all ros2 packages have been moved out from this file, reference to ros2 in this should be good to remove as well.

Reply:
It might be needed for the packages that are in ROS2 which are in common packages list

Reply:
Yeah we need the flag for ROS2

Reply:
Actually I realized we are not building anything but just downloading them, my bad. I removed the build-tool tags